packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/CozyMarkerBuilder.m

```diff
@@ -0,0 +1,235 @@
+//
+//  CozyMarkerBuilder.m
+//  google_maps_flutter_ios
+//
+//  Created by Luiz Carvalho on 16/02/23.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreText/CoreText.h>
+#import "CozyMarkerBuilder.h"
+
+
+@implementation CozyMarkerBuilder
+
+
+- (instancetype)initCozy {
+    self = [super init];
+    if(self) {
+    _cache = [[NSCache alloc] init];
```

Comment:
i didn't really found the need to decouple the caching in this class because the code usage itself is pretty small, but if proven needed we can change it.

Reply:
decoupling is more about context and responsibilities than code size, at least imo. I think it's ok to keep it that way because of the need of another refactor tho